certified-connectors/Connect Bridge/apiDefinition.swagger.json

```diff
@@ -102,6 +102,18 @@
         "description": "This action creates a new connection so that it can be used to execute future queries.",
         "operationId": "CreateConnection",
         "parameters": [
+          {
+            "name": "format",
+            "in": "query",
+            "required": true,
```

Comment:
Please confirm that the existing flows for this connector will not break from this change. 

The required parameter with default value has been added. ValuePath: operation/CreateConnection/parameter/format Source

Reply:
I confirm that the existing flows will not break from this change.

Actually this is a workaround because the policy stopped working for some reason.